### DISCUSSION TITLE
Section 1: explain coordinated versus affiliated packages

### DISCUSSION THREAD

**Discussion by bmerry** [astropy/astropy-v5.0-paper#80] [General comments] [answered]
I have a vague idea of the difference between astropy coordinated packages versus astropy affiliated packages, but in general readers might not, and it might be useful to explain the difference.

**Comment by pllim** [comment]
I agree!

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
